README.md

```diff
@@ -104,6 +104,7 @@ JAEGER_DISABLED _(not recommended)_ | Instructs the Configuration to return a no
 JAEGER_AGENT_HOST | The hostname for communicating with agent via UDP
 JAEGER_AGENT_PORT | The port for communicating with agent via UDP
 JAEGER_ENDPOINT | The traces endpoint, in case the client should connect directly to the Collector, like http://jaeger-collector:14268/api/traces
+JAEGER_PROPAGATION | The propagation format used by the tracer. Supported values are jaeger and w3c
```

Comment:
I would suggest not doing this in this PR, because supporting more than one format simultaneously is actually non-trivial, and not even clearly defined semantically, e.g.
  * do we include one or both formats in the outgoing requests?
  * what to do if incoming request has both formats? Especially if they are conflicting?